wrappers/huaweicloudkms/huaweicloudkms.go

```diff
@@ -0,0 +1,337 @@
+package huaweicloudkms
+
+import (
+	"context"
+	"crypto/tls"
+	"encoding/base64"
+	"fmt"
+	"os"
+	"sync/atomic"
+
+	"github.com/hashicorp/go-cleanhttp"
+	wrapping "github.com/hashicorp/go-kms-wrapping"
+	"github.com/huaweicloud/golangsdk"
+	huaweisdk "github.com/huaweicloud/golangsdk/openstack"
+	kmsKeys "github.com/huaweicloud/golangsdk/openstack/kms/v1/keys"
+)
+
+// These constants contain the accepted env vars; the Vault one is for backwards compat
+const (
+	EnvHuaweiCloudKMSWrapperKeyID   = "HUAWEICLOUDKMS_WRAPPER_KEY_ID"
+	EnvVaultHuaweiCloudKMSSealKeyID = "VAULT_HUAWEICLOUDKMS_SEAL_KEY_ID"
```

Comment:
You don't need the VAULT one as this never existed directly in Vault.

Reply:
Done